File: java-sdk/src/main/java/com/wikiglobal/wikibroker/openapi/common/enums/CustomHeaders.java
package com.wikiglobal.wikibroker.openapi.common.enums;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.experimental.Accessors;

@AllArgsConstructor
public enum CustomHeaders {
    ApiKey("X-Api-Key"),
    TimeStamp("X-Timestamp"),
    Nonce("X-Nonce"),
    Signature("X-Signature");

    @Getter
    @Accessors(fluent = true)
    private final String value;
}

File: php-sdk/src/Enum/CustomHeaders.php
<?php

declare(strict_types=1);

namespace WikibrokerOpenapiSdk\Enum;

enum CustomHeaders: string
{
    case ApiKey = 'X-Api-Key';
    case Timestamp = 'X-Timestamp';
    case Nonce = 'X-Nonce';
    case Signature = 'X-Signature';
}
